source/docs/software/advanced-control/state-space/state-space-flywheel-walkthrough.rst

```diff
@@ -1,4 +1,123 @@
-State-Space Controller Examples
-===============================
+State-Space Controller Walkthrough
+==================================
 
+.. note:: Before following this tutorial, readers are recommended to have read an :ref:`docs/software/advanced-control/state-space/state-space-intro:Introduction to state-space control`.
 
+The goal of this tutorial is to provide "end-to-end" instruction on implementing a state-space controller on a flywheel.  By following this tutorial, readers will learn how to:
+
+1. Create an accurate state-space model of a flywheel using :term:`system identification` or CAD software
+2. Implement a Kalman Filter to laglessly filter encoder velocity measurements.
+3. Implement a :ref:`LQR <docs/software/advanced-control/state-space/state-space-intro:The Linear-Quadratic Regulator>` feedback controller which, when combined with model-based feedforward, will generate voltrage :term:input:s to drive the flywheel to a :term:`reference`.
```

Comment:
Either remove periods at the end of all bullet points or add period to the first bullet point for consistency.